static/_redirects

```diff
@@ -367,7 +367,7 @@
 /docs/user-guide/kubectl-overview/     /docs/reference/kubectl/overview/
 /docs/user-guide/kubectl/     /docs/reference/generated/kubectl/kubectl-options/
 /docs/user-guide/kubectl-conventions/     /docs/reference/kubectl/conventions/
-/docs/user-guide/kubectl-cheatsheet/     /docs/reference/kubectl/cheatsheet/
```

Comment:
It's OK to leave this line in. I don't think it's a good idea to remove it. Instead, I recommend keeping line 370 from the original and then _adding_ an extra redirect.